lancero_source.go

```diff
@@ -123,6 +129,14 @@ func (ls *LanceroSource) Configure(config *LanceroSourceConfig) error {
 		config.AvailableCards = append(config.AvailableCards, k)
 	}
 	sort.Ints(config.AvailableCards)
+
+	// Ignore nsamp if not positive
```

Comment:
lets not do any ignore, just throw an error

Reply:
I was thinking we want to be able to configure the Lancero without specifying NSAMP, which this accomplishes. 

However, I guess what we really want is the opposite: we eventually want Cringe to be able to set NSAMP without changing any other configurations. Of course, this can be solved by at least two other approaches:

1. Make an RPC command that sets only the NSAMP value and doesn't touch other Lancero configurations. Cringe will use that.
1. Cringe "listens" to the ZMQ status port, learns the current Lancero config, and sends back an RPC request using that config (but with the NSAMP updated, obviously).

Maybe I'll change this to an error, as you suggest.